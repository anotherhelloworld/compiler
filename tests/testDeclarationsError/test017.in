const
    a: type = 10;
    b = 20;