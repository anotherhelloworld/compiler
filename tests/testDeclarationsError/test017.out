Unknown typename "type" Line 2 Column 8
